src/scales/scale.linearbase.js

```diff
@@ -25,7 +25,7 @@ function generateTicks(generationOptions, dataRange) {
 	var niceMax = Math.ceil(dataRange.max / spacing) * spacing;
 
 	// If min, max and stepSize is set and they make an evenly spaced scale use it.
-	if (generationOptions.min && generationOptions.max && generationOptions.stepSize) {
+	if (generationOptions.min !== undefined && generationOptions.max !== undefined && generationOptions.stepSize !== undefined) {
```

Comment:
is it wise to also check for `NaN`, infinite values, or values of other types. Perhaps `if (!isNaN(generationOptions.min) && isFinite(generationOptions.min) ... )`

Reply:
I feel that if we were going to check for those values we should probably log an error or throw an exception or somehow tell the user they've passed in invalid values rather than just silently ignoring the parameters. And maybe we should have a single method to validate user options called in the constructor or something rather than placing that code here

But in any case I feel it's quite unlikely for a user to set their options to `Infinity`, so I'm not sure I would require it before merging this PR. It would be a nice thing to do and I wouldn't object to validating user options, but this seems like a positive change even without that further improvement, so I don't want to block this PR on it necessarily